src/components/m-col/index.js

```diff
@@ -7,9 +7,9 @@ import defineOnce from '../../js/define-once';
 import styles from './index.scss';
 
 const reWhiteSpace = /\s+/;
-const reSize = /\d+|(?:xs|sm|md|lg|xl)(?:-(?:\d+|auto|))?/;
-const reOrder = /\d+|first|last|(?:xs|sm|md|lg|xl)-(?:\d+|first|last)/;
-const reOffset = /\d+|(?:xs|sm|md|lg|xl)(?:-\d+)?/;
+const reSize = /\d+|auto|(?:xs|sm|md|lg|xl|xxl)(?:-(?:\d+|auto|))?/;
```

Comment:
@jojackpot 
@Ben-Zahler 
This commit https://github.com/axa-ch/patterns-library/commit/2cdb094e16349ce12f77ccc0e3b3b89e066d5729 introduced a new `xxl` breakpoint, but without a `max-width` setting? Is this indented?

Reply:
@all if `xxl` is not indented to be available generically, then above commit should be undone and just implemented for the needed component 🤔

Reply:
yes, if is not indented generically.

Reply:
max-width may have been missed, please add it if required.
I cannot decide if xxl should be implemented only on the specified component, if you change anything, please make sure that the code is backward compatible with the current AEM implementation.

Reply:
@Ben-Zahler thanks for your answer.
I see, we should clarify that, @jojackpot whats the think with `xxl`, it's not an official breakpoint yet

Reply:
one more thing: according to Jonas, xxl is an official breakpoint in the AXA group styleguide. That's why we added it as a generic breakpoint

Reply:
Thanks @Ben-Zahler 
In that case the official style-guide page is out-of-date or it's a swiss customisation 🤔 
https://design.axa.com/web-guidelines/structure#grids

Reply:
Awesome feedback by @jojackpot 
> Die Auflösung 1920 x 1080 ist die häufigste Screen Resolution in der Schweiz: http://gs.statcounter.com/screen-resolution-stats/all/switzerland . Aus dem Grund wäre es sinnvoll diese in die Patterns Library zu übernehmen.